### ISSUE TITLE
zig build -Doptimize=ReleaseSafe not working

### ISSUE THREAD

**Issue by noahgsolomon** [fubark/zig-v8#16]
After running zig build get-v8, running  zig build -Doptimize=ReleaseSafe returns the following (I am using zig 0.12.0):

(the only thing in v8-build/x86_64-linux/release/ninja is "args.gn")

➜ zig build -Doptimize=ReleaseSafe
install
└─ v8
   └─ cp failure
error: FileNotFound
ninja: Entering directory `v8-build/x86_64-linux/release/ninja'
ninja: fatal: chdir to 'v8-build/x86_64-linux/release/ninja' - No such file or d
install
└─ v8
   └─ run tools/linux-amd64/ninja failure
error: the following command exited with error code 1:
tools/linux-amd64/ninja -C v8-build/x86_64-linux/release/ninja c_v8 
ERROR Unable to load "/home/noahsolomon/zig-v8/gclient/v8/zig/BUILD.gn".
install
└─ v8
   └─ run tools/linux-amd64/gn failure
error: the following command exited with error code 1:
tools/linux-amd64/gn --root=gclient/v8 --root-target=//zig --dotfile=.gn gen v8-
arget_os="linux" target_cpu="x64" is_debug=false symbol_level=0 is_official_buil
nable_handle_zapping=false cxx_use_ld="lld" 
Build Summary: 1/6 steps succeeded; 3 failed (disable with --summary none)
install transitive failure
└─ v8 transitive failure
   ├─ cp failure
   ├─ run tools/linux-amd64/gn failure
   └─ run tools/linux-amd64/ninja failure
error: the following build command failed with exit code 1:
/home/noahsolomon/zig-v8/zig-cache/o/1a50bf598b54becce26ba4a20a043d52/build /opt
/home/noahsolomon/zig-v8/zig-cache /home/noahsolomon/.cache/zig --seed 0x68757fc
fe

**Comment by fubark** [comment]
Did you run `zig build get-tools`?

**Comment by noahgsolomon** [comment]
> Did you run `zig build get-tools`?

Yes. I did this for zig version 12, 13 and 14 none of which worked

**Comment by jonathannen** [comment]
~~@noahgsolomon I've got a small PR here https://github.com/fubark/zig-v8/pull/17 - I'd be interested if this works for your case!~~ It's merged :)

It is also worth deleting the `.zig-cache` - after my fix I got a very similar error to you and removing the cache fixed it.
